_sass/blog.scss

```diff
@@ -15,4 +15,8 @@
   color: var(--dc-gray);
   text-transform: uppercase;
   font-size: 1rem;
+}
+
+img {
```

Comment:
Can we restrict this to blog pages? There are other images with text (e.g. tutorials) which would be too small to read if we restrict the width.

Reply:
done